README.md

```diff
@@ -81,39 +86,87 @@ envctl version
-    *   Prints a summary and instructions for MCP.
+    *   Launches an interactive terminal UI
+    *   Logs into `myinstallation` via `tsh kube login myinstallation`.
+    *   Sets the current `kubectl` context to `teleport.giantswarm.io-myinstallation`.
+    *   Starts port-forwarding for Prometheus (MC) on `localhost:8080`, Grafana (MC) on `localhost:3000`, and Alloy Metrics (MC) on `localhost:12345`.
+    *   Displays cluster health and connection status
+    *   Allows management of port-forwards and contexts
 
 2.  **Connect to a Management and Workload Cluster:**
 
     ```bash
-    envctl connect wallaby plant-cassino-prod
+    envctl connect myinstallation myworkloadcluster
     ```
 
-    *   Logs into `wallaby` via `tsh kube login wallaby`.
-    *   Logs into the *full* workload cluster name (`wallaby-plant-cassino-prod`) via `tsh`.
-    *   Sets the current `kubectl` context to the *full* workload cluster name (`teleport.giantswarm.io-wallaby-plant-cassino-prod`).
-    *   Starts port-forwarding for Prometheus using the *management cluster* context (`teleport.giantswarm.io-wallaby`) in the background.
-    *   Starts port-forwarding for Alloy metrics using the *workload cluster* context (`teleport.giantswarm.io-wallaby-plant-cassino-prod`) with ports 12345:12345.
+    *   Logs into `myinstallation` via `tsh kube login myinstallation`.
+    *   Logs into the *full* workload cluster name (`myinstallation-myworkloadcluster`) via `tsh`.
+    *   Sets the current `kubectl` context to the *full* workload cluster name (`teleport.giantswarm.io-myinstallation-myworkloadcluster`).
+    *   Starts port-forwarding for Prometheus using the *management cluster* context (`teleport.giantswarm.io-myinstallation`) to `localhost:8080`.
+    *   Starts port-forwarding for Grafana using the *management cluster* context (`teleport.giantswarm.io-myinstallation`) to `localhost:3000`.
+    *   Starts port-forwarding for Alloy metrics using the *workload cluster* context (`teleport.giantswarm.io-myinstallation-myworkloadcluster`) to `localhost:12345`.
     *   Prints a summary and instructions for MCP.
 
+## Terminal User Interface 🖥️
+
+When running `envctl connect`, the Terminal User Interface (TUI) provides a visual dashboard to monitor and control your connections:
+
+![envctl TUI overview](docs/images/tui-overview.png)
+
+### Key Features
+
+- **Cluster Status Monitoring**: View real-time health status of both management and workload clusters
+- **Port Forward Management**: Monitor active port forwards with status indicators
+- **Log Viewer**: View operation logs directly in the terminal
+- **Keyboard Navigation**: Easily navigate between panels with Tab/Shift+Tab
+- **Dark Mode Support**: Toggle between light and dark themes with 'D' key
+
+### Keyboard Shortcuts
+
+| Key          | Action                                   |
+|--------------|------------------------------------------|
+| Tab          | Navigate to next panel                   |
```

Comment:
The README mentions Prometheus being available at `/prometheus` path, but this isn't configured in the port forwarding setup. Either update the documentation or ensure the path is properly configured in the port forwarding.